plugins/lookup/bitwarden.py

```diff
@@ -232,6 +261,33 @@ def get_field(self, field, search_value, search_field="name", collection_id=None
 
         return field_matches
 
+    def get_attachment(self, attachment_name, search_value, search_field="name", collection_id=None, organization_id=None):
+        """Download attachment from records whose search_field match search_value.
+        Returns a list of attachment contents (as raw bytes converted to text) for each matching item.
+        """
+        matches = self._get_matches(search_value, search_field, collection_id, organization_id)
+
+        if not matches:
+            raise AnsibleError(f"No item found matching {search_field}={search_value}")
+
+        attachment_contents = []
+        for match in matches:
+            item_id = match.get('id')
+            if not item_id:
+                raise AnsibleError(f"Item {match.get('name', 'unknown')} has no ID")
+
+            try:
+                params = ['get', 'attachment', attachment_name, '--itemid', item_id, '--raw']
+                out, err = self._run(params)
```

Comment:
`out` and `err` are unicode strings; the output of the CLI is parsed as UTF-8. Is this what is meant by "raw format"?

(Can Bitwarden actually store binary files, or must all files be UTF-8 encoded text?)